### PR TITLE
Add simple Pythom examples

### PR DESCRIPTION
**Pull request by kiyoto**

### CODE REVIEW

File: td/python-scripting/pandas-df/scripts/examples.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Because we do not allow custom image currently, this is how you can add 3rd party
+# libraries instead
+import sys, os
+sys.path.append('/home/td-user/.local/lib/python3.6/site-packages')
```
Comment (chezou) [inline td/python-scripting/pandas-df/scripts/examples.py]: [nits] Using `sys.version_info` would be more robust for future version changes.

```suggestion
py_ver = f'{sys.version_info.major}.{sys.version_info.minor}'
sys.path.append(f'/home/td-user/.local/lib/python{py_ver}/site-packages')
```

File: td/python-scripting/pandas-df/scripts/examples.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+os.system(f"{sys.executable} -m pip install --user pandas-td")
+
+import pandas_td as td
+import tdclient
+
+con = td.connect(apikey=os.environ['TD_API_KEY'], endpoint=os.environ['TD_API_SERVER'])
+
+def read_td_table(database_name, table_name, engine_name = 'presto', limit=1000):
+    engine = td.create_engine("{}:{}".format(engine_name, database_name), con=con)
```
Comment (chezou) [inline td/python-scripting/pandas-df/scripts/examples.py]: [nits] how about using f-string?

```suggestion
    engine = td.create_engine(f"{engine_name}:{database_name}", con=con)
```

File: td/python-scripting/pandas-df/scripts/examples.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    df = td.read_td('SELECT * FROM {} LIMIT {}'.format(table_name, limit), engine)
```
Comment (chezou) [inline td/python-scripting/pandas-df/scripts/examples.py]: ditto

```suggestion
    df = td.read_td(f'SELECT * FROM {table_name} LIMIT {limit}', engine)  
```

File: td/python-scripting/pandas-df/scripts/examples.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+def read_td_table(database_name, table_name, engine_name = 'presto', limit=1000):
+    engine = td.create_engine("{}:{}".format(engine_name, database_name), con=con)
+    df = td.read_td('SELECT * FROM {} LIMIT {}'.format(table_name, limit), engine)  
+    print(df)
+    
+def write_td_table(database_name, table_name):
+    import pandas as pd
+    import random
+    # TODO TD client, check for table's existence
+    engine = td.create_engine("presto:{}".format(database_name), con=con)
```
Comment (chezou) [inline td/python-scripting/pandas-df/scripts/examples.py]: ditto

```suggestion
    engine = td.create_engine(f"presto:{database_name}", con=con)
```

File: td/python-scripting/pandas-df/scripts/examples.py [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        tdc.create_database(database_name)
+    except AlreadyExistsError:
+        pass
+    
+    try:
+        tdc.create_log_table(database_name, table_name)
+    except AlreadyExistsError:
+        pass
+    
+    table_path = "{}.{}".format(database_name, table_name)
```
Comment (chezou) [inline td/python-scripting/pandas-df/scripts/examples.py]: ditto

```suggestion
    table_path = f"{database_name}.{table_name}"
```

File: td/python-scripting/simple/scripts/examples.py [hunk 6]
```diff
@@ -0,0 +1,14 @@
+# Note that "meg" parameter comes from the `msg` field in simple.dig
+# This is how parameters can be passed in
+def print_arg(msg):
+    print("Message is {}".format(msg))
```
Comment (chezou) [inline td/python-scripting/simple/scripts/examples.py]: ditto
```suggestion
    print(f"Message is {msg}")
```

File: td/python-scripting/simple/scripts/examples.py [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+    
+    
+def print_env():
+    import os
+    print("Env Var is {}".format(os.environ['MY_ENV_VAR']))
```
Comment (chezou) [inline td/python-scripting/simple/scripts/examples.py]:
```suggestion
    print(f"Env Var is {os.environ['MY_ENV_VAR']}")
```

**Comment by kiyoto** [comment]
Thank you @chezou, very good feedback. (Had to accept individual suggestions, the “add suggestion to a batch” button was grayed out)

If you want me to merge these commits, let me know.

**Comment by chezou** [comment]
Looks good to me with notification around py> operator is not GA yet.

**Comment by kiyoto** [comment]
@chezou Added a disclaimer + squashed commits.

**Comment by chezou** [comment]
@kiyoto ship it 👍

**Comment by toru-takahashi** [comment]
Thank you!

**Pull request merged**